### ISSUE TITLE
module_install don't work

### ISSUE THREAD

**Issue by beherit** [jabber-at/ejabberd#5]
I have problem with building (installing) modules, module_install shows an error, e.g.:

~~~
root@virtualbox:~# ejabberdctl module_install mod_admin_extra
:23: can't find include file "xml.hrl"
:430: record xmlel undefined
:470: type xmlel() undefined
src/mod_admin_extra.erl:980: record xmlel undefined
src/mod_admin_extra.erl:1432: record xmlel undefined
:23: can't find include file "xml.hrl"
:430: record xmlel undefined
:470: type xmlel() undefined
Error: {compilation_failed,"src/mod_admin_extra.erl"}
~~~

Problem doesn't exists when I build ejabberd from processone source or installer - only when I use your repo problem exists.

**Comment by beherit** [comment]
I found solution here:

https://www.ejabberd.im/node/23062

Should to compile ejabberd with NO_EXT_LIB flag?

**Comment by mathiasertl** [comment]
This appears to be an upstream problem and probably won't be fixed until the next version. Sorry.

**Comment by beherit** [comment]
It's problem with your package XML. Check it out properly.

**Comment by beherit** [comment]
Ejabberd 15.03 from processone installer work properly - it's the same upstream.

**Comment by beherit** [comment]
Please reopen this issue :(

**Comment by mathiasertl** [comment]
It is related to the fact that ejabberd is installed with separate packages. I talked with an upstream developer and it is supposed to be fixed in 15.04. 

PS: Please don't randomly accuse me of "not checking it out properly" in the future. I consider this quite rude. Thank you.

**Comment by beherit** [comment]
I updated ejabberd from Your repo on test machine (I had to use wheezy-backports because You picked up version of e.g. erlang-base) and the problem still exists.
